content/java/concepts/strings/terms/matches/matches.md

```diff
@@ -0,0 +1,50 @@
+---
+Title: '.matches()'
+Description: 'The method checks whether a string matches a given regular expression.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Strings'
+  - 'Methods'
+CatalogContent:
+  - 'learn-java'
+  - 'paths/computer-science'
+---
+
+The **`.matches()`** method checks whether a string matches a given regular expression, returning True or False.
+
+## Syntax
+
+```pseudo
+string.matches(regex)
+```
+
+The `.matches()` method takes a single parameter:
+  - regex - a regular expression that the string is compared to.
+ 
+## Return Value
```

Comment:
Let's remove this header so the below is in the Syntax section.

Reply:
Removed